### PR TITLE
updating README in preparation for matlab-actions/run-command@v2

### PR DESCRIPTION
**Pull request by mw-hrastega**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,15 +17,15 @@ jobs:
...
       - name: Run script
-        uses: matlab-actions/run-command@v1
+        uses: matlab-actions/run-command@v2
         with:
           command: myscript
 ```
 
 ### Run MATLAB Commands on GitHub-Hosted Runner
-Before you run MATLAB code or Simulink models on a GitHub-hosted runner, first use the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action. The action sets up your specified MATLAB release (R2020a or later) on a Linux&reg; virtual machine. If you do not specify a release, the action sets up the latest release of MATLAB.
+Before you run MATLAB code or Simulink models on a [GitHub-hosted runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners), first use the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action. The action sets up your specified MATLAB release (R2020a or later) on a Linux&reg;, Windows&reg;, or macOS virtual machine. If you do not specify a release, the action sets up the latest release of MATLAB.
```
Comment (mcafaro) [inline README.md:28]: I believe `setup-matlab@v2` is R2020b or later.

Comment (mcafaro) [inline README.md:28]: macOS is a registered trademark. https://www.apple.com/legal/intellectual-property/trademark/appletmlist.html

File: README.md [hunk 2]
```diff
@@ -1,16 +1,12 @@
...
-
-- To use a GitHub-hosted runner, you must include the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action in your workflow to set up MATLAB on the runner. Currently, this action is available only for public projects. It does not set up transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;.
+The [Run MATLAB Command](#run-matlab-command) action enables you to execute MATLAB&reg; scripts, functions, and statements on a self-hosted or GitHub&reg;-hosted runner. The runner uses the topmost MATLAB version on the system path to execute your workflow.
 
 ## Examples
 Use the **Run MATLAB Command** action to run MATLAB scripts, functions, and statements. You can use this action to flexibly customize your test run or add a step in MATLAB to your workflow.
 
 ### Run MATLAB Script on Self-Hosted Runner
-Use a self-hosted runner to run the commands in a file named `myscript.m` in the root of your repository.
+Use a [self-hosted runner](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners) to run the commands in a file named `myscript.m` in the root of your repository.
```
Comment (mcafaro) [inline README.md:9]: It would be nice to get the "R2013b or later" back in here somewhere.

File: README.md [hunk 3]
```diff
@@ -1,16 +1,12 @@
...
 
-The [Run MATLAB Command](#run-matlab-command) action enables you to execute MATLAB&reg; scripts, functions, or statements on a [self-hosted](https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners) or [GitHub&reg;-hosted](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners) runner:
-
-- To use a self-hosted runner, you must set up a computer with MATLAB (R2013b or later) as your runner. The runner uses the topmost MATLAB version on the system path to execute your workflow.
-
-- To use a GitHub-hosted runner, you must include the [Setup MATLAB](https://github.com/matlab-actions/setup-matlab/) action in your workflow to set up MATLAB on the runner. Currently, this action is available only for public projects. It does not set up transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;.
+The [Run MATLAB Command](#run-matlab-command) action enables you to execute MATLAB&reg; scripts, functions, and statements on a self-hosted or GitHub&reg;-hosted runner. The runner uses the topmost MATLAB version on the system path to execute your workflow.
```
Comment (mcafaro) [inline README.md:3]: "to execute your workflow" This action is not necessarily your entire workflow.

```suggestion
The [Run MATLAB Command](#run-matlab-command) action enables you to execute MATLAB&reg; scripts, functions, and statements on a self-hosted or GitHub&reg;-hosted runner. The action uses the topmost MATLAB version on the system path.
```

File: README.md (hunk 1 above)
Comment (mw-hrastega) [reply README.md]: Fixed.

Comment (mw-hrastega) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (mw-hrastega) [reply README.md:9]: Decided to leave it out after discussing with Mark. Will add the information if we have customer requests.

**Review by mcafaro — APPROVED** [review]

**Review by davidbuzinski — APPROVED** [review]

**Pull request merged**